src/main.js

```diff
@@ -1827,12 +1856,18 @@ const DiscoveryFeedApplication = new Lang.Class({
         if (this._debugWindow)
             return;
 
-        let gdkWindow = this._window.get_window();
-        gdkWindow.set_events(gdkWindow.get_events() |
-                             Gdk.EventMask.FOCUS_CHANGE_MASK);
-        this._window.connect('focus-out-event', Lang.bind(this, function() {
-            this.hide();
-            return false;
+        // Connecting to 'draw' here is not ideal, however it seems like
+        // the 'realize' signal is not fired for GtkWindow when it gets
+        // mapped
+        this._signals.track(this._window, 'draw', Lang.bind(this, function() {
```

Comment:
per @cosimoc 

> Pretty sure this is because we already call realize() on a children before calling into this: 8b25b4d#diff-1c90ff38a08209f9ebd4d05d1e43358eR1726

per @smspillaz

> Ah interesting. Should we connect to the realize signal and make that call below instead then?

per @cosimoc 

> I actually don't think there's any change needed in how we handle the connection to focus-out-event. Probably this was required when you first wrote the PR, and we still depended on Wnck, but not right now. I'd just drop that commit.

It seems like my original comment disappeared :( But I clarified why we were doing this in the commit message.